multilep/test/multilep.py

```diff
@@ -104,6 +105,34 @@ def getVal(arg):
 
 yy = '17' if is2017 or is2018 else '16'
 
+#
+#Latest 94X tau ID
+#
+from heavyNeutrino.multilep.runTauIdMVA import *
+na = TauIDEmbedder(process, cms, # pass tour process object
+    debug=True,
+    toKeep = ["2017v2", "newDM2017v2"] # pick the one you need: ["2017v1", "2017v2", "newDM2017v2", "dR0p32017v2", "2016v1", "newDM2016v1"]
+)
+na.runTauID()
+
+byIsolationMVArun2017v2DBoldDMwLTraw2017 = cms.string('byIsolationMVArun2017v2DBoldDMwLTraw2017'),
```

Comment:
Is this line (and all the following lines) really useful? I don't see how it is used.

Reply:
I am not 100% sure about this. I more or less tried to follow the instructions of the tau POG in [this twiki entry](https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuidePFTauID#Rerunning_of_the_tau_ID_on_M_AN1)

Reply:
The twiki is not very clear, but according to the presentation it has attached these parts rather belong somewhere else, like here:
https://github.com/greyxray/TauAnalysisTools/blob/CMSSW_9_4_X_tau-pog_RunIIFall17/TauAnalysisTools/test/produceTauIdMVATrainingNtupleMiniAOD_cfg.py#L207#L268

Reply:
I don't think the lines are needed, because in the example these lines are in a tauIDdiscriminators PSet, which is used in the following way:
1) each of those strings are stored in a tauIdDiscrEntries_ vector
https://github.com/greyxray/TauAnalysisTools/blob/CMSSW_9_4_X_tau-pog_RunIIFall17/TauAnalysisTools/plugins/TauIdMVATrainingNtupleProducer.cc#L77-L84
2) this vector is then used to automatically add these strings to their tuple:
https://github.com/greyxray/TauAnalysisTools/blob/CMSSW_9_4_X_tau-pog_RunIIFall17/TauAnalysisTools/plugins/TauIdMVATrainingNtupleProducer.cc#L184-L187
3) and then the value is filled
https://github.com/greyxray/TauAnalysisTools/blob/CMSSW_9_4_X_tau-pog_RunIIFall17/TauAnalysisTools/plugins/TauIdMVATrainingNtupleProducer.cc#L623-L629

but you instead retrieves the value in this way:
https://github.com/GhentAnalysis/heavyNeutrino/pull/56/files#diff-cb0159fc9d53726859648ee943978eaeR312 using this tau.tauID(string) formalism

Reply:
anyway I think their step 3 in the twiki is useless :-)

Reply:
Ah yes, I see, I'll just remove them then